apps/charterafrica/src/utils/fetchJson.js

```diff
@@ -10,6 +11,7 @@ export class FetchError extends Error {
     this.message = message;
     this.data = data;
     this.status = status;
+    Sentry.captureMessage(message);
```

Comment:
Weird seeing this here. It should be on `catch` where this is used (if the situation calls for it to be captured).